README.md

```diff
@@ -1,24 +1,169 @@
+
+## 🚀 Live Version <a name="live-demo"></a>
+
+- [Comming soon]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation -->
+
+## 🚀 Presenation video <a name="live-demo"></a>
+
+- [Video](https://drive.google.com/file/d/1Iu-ubjm5I7dyXyw5yfJep3XAd3SjsGuj/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Visual Studio Code and Installed Rails.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash command
+$ git clone $ git clone https://github.com/iambenkis/Budget-app-with-rails.git
+```
+
+### Install
+
+Install this project with:
+
+```bash command
+you@your-Pc-name:~$ cd <budget-app>
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash command
+` $ rails server`
+```
+
+### Run tests
+
+To run tests, run the following command:
+```bash command
+ $ rspec
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Benjamin Kisenge**
+
+- GitHub: [@iambenkis](https://github.com/iambenkis)
+- LinkedIn: [Benjamin Kisenge ](https://www.linkedin.com/in/benjamin-kisenge/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add some tests]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give as a star! ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[How I can run this project?]**
+
+  - [After cloning repository, run rails server.]
+
+- **[Can I add likes on post?]**
+
+  - [You can add likes and comments on a post.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
[OPTIONAL] This note is only instructions for you to set up the license. I think you should remove it to keep your readme looking professional.